docs/installation.rst

```diff
@@ -4,79 +4,70 @@
 Installation
 ************
 
-Requirements
-============
+The SunPy project maintains a range of libraries that leverage the wider ecosystem of scientific Python packages for solar physics.
 
-sunpy requires Python 3.8 or higher.
+Installing a SunPy library
```

Comment:
I think we should leave this bit of the docs to just installing `sunpy` core. I've never heard the phrase "SunPy library" before.

Reply:
I think my original idea was to enable this to be generic enough that we don't have to write this page for every package under the sunpy project. 

The title is bad, I will admit to that.

Reply:
I am not sure I agree, I don't want to have to maintain 10 copies of this page for each of the sponsored packages, what do you think we are loosing by making this useful to all our packages.

Reply:
I think the specific language of "SunPy library" confused me. I think the way forward here is to make the installation instructions specific to sunpy core, but mention liberally throughout that this provides a base environment for intalling "other SunPy affiliated packages" (or whatever we're calling other packages these days)